sbi/neural_nets/embedding_nets/SC_embedding.py

```diff
@@ -0,0 +1,311 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Apache License Version 2.0, see <https://www.apache.org/licenses/>
+
+# This code is based on on the following three papers:
```

Comment:
I think it is good to keep the papers and git repositories here, if the code is adapted from here.

However, please add at least the relevant papers for background knowledge to the docstring - as this will not be rendered in the documentation/website.